packages/mock-account-provider/app/routes/webhooks.ts

```diff
@@ -63,9 +63,9 @@ export async function handleOutgoingPaymentCompletedFailed(wh: WebHook) {
   const amtSend = payment['sendAmount'] as Amount
   const amtSent = payment['sentAmount'] as Amount
```

Comment:
I think `Amount`'s `value` type should be changed from `bigint` to `string`
https://github.com/interledger/rafiki/blob/603df09615a2f458227baf3e23f99b3a0b103921/packages/mock-account-provider/app/routes/webhooks.ts#L23

and/or maybe map could use a:
https://github.com/interledger/rafiki/blob/603df09615a2f458227baf3e23f99b3a0b103921/packages/backend/src/open_payments/amount.ts#L13-L19